legacy_publisher/dita_publisher.py

```diff
@@ -1,5 +1,71 @@
+import json
+import random
+import os 
 
-def publish_regions(map_url, regions, target_folder):
-    print('outputting regions', map_url, target_folder)
-    for reg in regions:
-        print("outputting region", reg.shape, reg.url, reg.coords)
\ No newline at end of file
+from xml.dom import minidom
+from os.path import dirname, abspath
+from legacyman_parser.utils.constants import DITA_REGIONS_EXPORT_FILE
+
+"""This module will handle post parsing enhancements for DITA publishing"""
+DITA_EXPORT_FILE = DITA_REGIONS_EXPORT_FILE
+
+random.seed(100)
```

Comment:
@brahnavank  - we don´t need or use random numbers, I guess this is a copy/paste error.  Please copy ideas, but not code.